### ISSUE TITLE
Desmodder on Desmos calculator embed

### ISSUE THREAD

**Issue by RannyBergamotte** [DesModder/DesModder#414]
Would be nice to be able to used Desmodder on the embed of certain sites. I know that orginally it was decided against so that on student.desmos.com couldn't cheat or whatnot, but would be extremely nice for https://teth-main.github.io/GraTeX/ for example. I'm sure in the future there might be other sites that could use this, that could be put on a whitelist or something like that.

**Comment by FenyLabs** [comment]
Unfortunately I believe this would require permissions on all sites, but it would definitely be possible.

**Comment by jared-hughes** [comment]
I agree, this would be nice! It's more of a technical issue than whitelisting a few sites because DesModder is unfortunately tightly integrated with the loading process of desmos.com/calculator specifically, and other sites use the API, which is different enough from desmos.com/calculator to require substantial work to get going.

Enabling permissions to access all sites is not a big issue
